README.md

```diff
@@ -678,147 +811,141 @@ extension Subprocess.Execution {
+        environment: Environment = .inherit,
+        workingDirectory: FilePath? = nil,
+        platformOptions: PlatformOptions = .default
+    )
 }
 
-extension Subprocess.Configuration : CustomStringConvertible, CustomDebugStringConvertible {}
+extension Configuration : CustomStringConvertible, CustomDebugStringConvertible {}
 ```
 
 **Note:** the `.workingDirectory` property defaults to the current working directory of the calling process.
 
 
-### `Subprocess.StandardInputWriter`
+### `StandardInputWriter`
 
-`StandardInputWriter` provides developers with direct control over writing to the child process's standard input. Similar to the `Subprocess` object itself, developers should use the `StandardInputWriter` object passed to the `body` closure, and this object is only valid within the body of the closure.
+`StandardInputWriter` provides developers with direct control over writing to the child process's standard input. Similar to the `Execution` object, developers should use the `StandardInputWriter` object passed to the `body` closure, and this object is only valid within the body of the closure.
 
 **Note**: Developers should call `finish()` when they have completed writing to signal that the standard input file descriptor should be closed.
 
 ```swift
-extension Subprocess {
-    /// A writer that writes to the standard input of the subprocess.
-    @available(FoundationPreview 0.4, *)
-    @available(iOS, unavailable)
-    @available(tvOS, unavailable)
-    @available(watchOS, unavailable)
-    public final actor StandardInputWriter {
-        /// Write a sequence of UInt8 to the standard input of the subprocess.
-        /// - Parameter sequence: The sequence of bytes to write.
-        public func write<SendableSequence: Sequence<UInt8> & Sendable>(
-            _ sequence: SendableSequence
-        ) async throws
-        /// Write a String to the standard input of the subprocess.
-        /// - Parameter sequence: The string write.
-        public func write(
-            _ string: some StringProtocol
-        ) async throws
-
-        /// Write a AsyncSequence of Data to the standard input of the subprocess.
-        /// - Parameter sequence: The sequence of bytes to write.
-        public func write<AsyncSendableSequence: AsyncSequence & Sendable>(
-            _ asyncSequence: AsyncSendableSequence
-        ) async throws where AsyncSendableSequence.Element == Data
-        /// Signal all writes are finished
-        public func finish() async throws
-    }
+/// A writer that writes to the standard input of the subprocess.
+@available(FoundationPreview 0.4, *)
+@available(iOS, unavailable)
+@available(tvOS, unavailable)
+@available(watchOS, unavailable)
+public final actor StandardInputWriter {
+    /// Write a sequence of UInt8 to the standard input of the subprocess.
+    /// - Parameter sequence: The sequence of bytes to write.
+    public func write<SendableSequence: Sequence<UInt8> & Sendable>(
```

Comment:
I'm still unsure about this - do we really want to support *any* sequence of `UInt8` or do we want to require a `Collection` instead so we know we can do a bulk operation? Even if we want to allow it, shouldn't we provide concrete overrides of this that take a `Collection` or some other known-contiguous types like `[UInt8]` to avoid the overhead of casting in the implementation?

Reply:
Explicit `ArrayInput` and `DataInput`.